lint-staged.config.js

```diff
@@ -1,14 +1,34 @@
 const escape = require('shell-quote').quote;
+const fs = require('fs');
 const isWin = process.platform === 'win32';
 
 module.exports = {
-  '**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}': filenames => {
+  '**/*{.css,.json,.md}': filenames => {
     const escapedFileNames = filenames
+      .filter(filename => fs.existsSync(filename))
       .map(filename => `"${isWin ? filename : escape([filename])}"`)
       .join(' ');
     return [
       `prettier --write ${escapedFileNames}`,
       `git add ${escapedFileNames}`
     ];
+  },
+  '**/*{.ts,.tsx}': filenames => {
+    const escapedFileNames = filenames
+      .filter(filename => fs.existsSync(filename))
+      .map(filename => `"${isWin ? filename : escape([filename])}"`)
+      .join(' ');
+    return [`tslint --fix ${escapedFileNames}`, `git add ${escapedFileNames}`];
```

Comment:
Based on what the `lint` call in package.json does I think updating this to: 
```suggestion
    return [
      `prettier --write ${escapedFileNames}`,
      `tslint --fix ${escapedFileNames}`,
      `git add ${escapedFileNames}`
    ];
```
and then removing `.ts,.tsx` from the next call would be more consistent.